src/app/page.tsx

```diff
@@ -11,14 +13,24 @@ const MAPLIST = LOCATIONS.map((location, index) => {
 });
 
 export default function Home() {
+  const [currentItemIndex, setCurrentItemIndex] = useState(0);
+
+  function increment() {
```

Comment:
Can we make this name more meaningful?

Reply:
Yes sir! Would `incrementActive` be more precise?

Reply:
I was thinking about `incrementItemIndex`. Wdyt?

Reply:
Lovin it!